dev-python/pkgconfig/python_pkgconfig-1.5.5.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="A Python module to interface with the pkg-config"
+DESCRIPTION="pkgconfig is a Python module to interface with the pkg-config command line tool for \
+Python 3.3+."
+HOMEPAGE="https://pypi.org/project/pkgconfig/"
+COPYRIGHT="201-2021 Mathias Vogelgesang"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://pypi.python.org/packages/source/p/pkgconfig/pkgconfig-$portVersion.tar.gz"
+CHECKSUM_SHA256="deb4163ef11f75b520d822d9505c1f462761b4309b1bb713d08689759ea8b899"
+SOURCE_DIR="pkgconfig-$portVersion"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	$portName = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+PYTHON_PACKAGES=(python3 python38 python39 python310)
+PYTHON_VERSIONS=(3.7 3.8 3.9 3.10)
+for i in "${!PYTHON_PACKAGES[@]}"; do
+pythonPackage=${PYTHON_PACKAGES[i]}
+pythonVersion=${PYTHON_VERSIONS[$i]}
+eval "PROVIDES_${pythonPackage}=\"\
+	${portName}_$pythonPackage = $portVersion\
+	\"; \
+REQUIRES_$pythonPackage=\"\
+	haiku\n\
+	cmd:pkg_config\n\
```

Comment:
After the discussion in #10737: I don't think that this requirement makes any sense here, because `cmd:pkg_config` is architecture-dependent (with `$secondaryArchSuffix`), and this is an "any" arch package. So anything that depends on this needs to depend on `cmd:pkg_config$secondaryArchSuffix` separately anyway (presuming that there are no "any" arch packages which provide pkg-config files and could be found with either pkg-config version).

Reply:
Makes sense now :) Mentions @OscarL

Reply:
I think @jmairboeck reply is the way to go, just nuke out "cmd:pkg_config" in here, no need to make python_pkgconfig arch aware.

Reply:
> Not sure that's what @jmairboeck is suggesting Begasus, unless I'm not following.

Yes, that is what I suggested. Having it like this is wrong for x86, and adding `cmd:pkg_config_x86` explicitly just for x86 is also wrong because then the package isn't architecture-independent any more.
 
However, I think the better solution would be to make this architecture-dependent then. The contents of the package would still be the same, but the metadata aren't.

In a way, the situation here is similar to alien_sdl, which I thought of making "any" but decided against it because it contains some configuration data with paths containing secondary arch suffixes, see https://github.com/haikuports/haikuports/blob/b2ff1616efbbd3ea0af1905d52155db82622d87b/dev-perl/alien_sdl/alien_sdl-1.446.recipe#L77-L78

Another alternative would be adding `SECONDARY_ARCHITECTURES="x86"` but keep "any", but as @pulkomandy wrote in https://github.com/haikuports/haikuports/pull/9436#discussion_r1623524876, "this doesn't make sense" (and he is right of course).